machine_learning_hep/plot_ml_std_comp.py

```diff
@@ -1,333 +0,0 @@
-######
-# Lc #
-######
-# Correct for branching ratios BR(Lc->pKpi) / BR(Lc->pK0s)
-compare_ml_std("LcpK0spp", "MBvspt_ntrkl", -1,
-               "data/std_results/HFPtSpectrum_LcpKpi_merged_20191010.root", 1./5.75, None, 0,
-               ["histoSigmaCorr"], ["histoSigmaCorr"])
-#compare_ml_std("LcpK0spp", "MBvspt_ntrkl", -1, "data/std_results/Lc_19_corryield.root",
-#               1./5.75 / SIGMA0, None, 1, ["histoYieldCorr"], ["corrYield_19"])
-#compare_ml_std("LcpK0spp", "MBvspt_ntrkl", -1, "data/std_results/Lc_1029_corryield.root",
-#               1./5.75 / SIGMA0, None, 2, ["histoYieldCorr"], ["corrYield_1029"])
-#compare_ml_std("LcpK0spp", "MBvspt_ntrkl", -1, "data/std_results/Lc_3059_corryield.root",
-#                1./5.75 / SIGMA0, None, 3, ["histoYieldCorr"], ["corrYield_3059"])
-
-# RATIOS
-###########
-# Lc / D0 #
-###########
-compare_ml_std_ratio("LcpK0spp", "D0pp", "MBvspt_ntrkl", -1,
-                     "data/std_results/HFPtSpectrum_LcpKpi_merged_20191010.root",
-                     "data/std_results/HFPtSpectrum_D0_merged_20191010.root", 1./5.75, None, None,
-                     0, ["histoSigmaCorr"], ["histoSigmaCorr"], ["histoSigmaCorr"])
-#compare_ml_std_ratio("LcpK0spp", "D0pp", "MBvspt_ntrkl", -1,
-#                     "data/std_results/Lc_19_corryield.root",
-#                     "data/std_results/D0_19_corryield.root", 1./5.75 / SIGMA0, 1. / SIGMA0,
-#                     None, 1, ["histoYieldCorr"], ["corrYield_19"], ["corrYield_19"])
-#compare_ml_std_ratio("LcpK0spp", "D0pp", "MBvspt_ntrkl", -1,
-#                     "data/std_results/Lc_1029_corryield.root",
-#                     "data/std_results/D0_1029_corryield.root", 1./5.75 / SIGMA0, 1. / SIGMA0,
-#                     None, 2, ["histoYieldCorr"], ["corrYield_1029"], ["corrYield_1029"])
-#compare_ml_std_ratio("LcpK0spp", "D0pp", "MBvspt_ntrkl", -1,
-#                     "data/std_results/Lc_3059_corryield.root",
-#                     "data/std_results/D0_3059_corryield.root", 1./5.75 / SIGMA0, 1. / SIGMA0,
-#                     None, 3, ["histoYieldCorr"], ["corrYield_3059"], ["corrYield_3059"])
-# PRELIM RESULTS
-compare_ml_std_ratio("LcpK0spp", "D0pp", "MBvspt_ntrkl", -1,
-                     "data/std_results/HFPtSpectrum_LcpKpi_2016_prel_20191010.root",
-                     "data/std_results/HFPtSpectrum_D0_2016_prel_20191010.root", 1./5.75,
-                     None, [(1, [1]), (2, [2, 3]), (3, [4, 5]), (4, [6]), (5, [7]), (6, [8])], 0,
-                     ["histoSigmaCorr"], ["histoSigmaCorr"], ["histoSigmaCorr"], "_prelim")
-
-###########
-# Ds / D0 #
-###########
-
-compare_ml_std_ratio("Dspp", "D0pp", "MBvspt_ntrkl", -1,
-                     "data/std_results/HFPtSpectrum_Ds_merged_20191010.root",
-                     "data/std_results/HFPtSpectrum_D0_merged_20191010.root", None, None, None,
-                     0, ["histoSigmaCorr"], ["hCrossSectionStatisticError"], ["histoSigmaCorr"])
-if FILES_NOT_FOUND:
```

Comment:
This part would also be useful to copy, it is not yet part of the other macro I see

Reply:
You are right, will add that